### ISSUE TITLE
Accessibility: A11y_Live Chat Widget_ Latest Bot Chat Window_AI4W:Ensures all ARIA attributes have valid values

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3054]
Copied from #1749533.

![1749533_A11y_Live Chat Widget_ Latest Bot Chat Window_AI4W](https://user-images.githubusercontent.com/1622400/77822145-57a05580-70ad-11ea-8ea8-a48ea410efa6.png)

“Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:

Application Name: Live Chat Widget

#URL: https://xxx.blob.core.windows.net/portals/democ2adaptivecard.html

Browser Versions: Microsoft Edge Version 80.0.361.54 (Official build) (64-bit)

Windows Version: Windows10

Chrome Version: Version 80.0.3987.132 (Official Build) (64-bit)

 

Repro Steps: ​​

 

Launch the application url : https://xxx.blob.core.windows.net/portals/democ2adaptivecard.html
Latest Bot icon Button get open and Click on it.
"Latest Bot Chat Window" screen get open.
Run AI4W and check the issue.
 

Issue: Issue: Ensures all ARIA attributes have valid values (aria-valid-attr-value - https://dequeuniversity.com/rules/axe/3.5/aria-valid-attr-value?application=msftAI)

 

 

Target application: Test Page For Adaptive Card - https://xxx.blob.core.windows.net/portals/democ2adaptivecard.html

 

 

Element path: #Microsoft_Omnichannel_LCWidget_Chat_Iframe_Window;#webchat__toaster__list__vo8ez

 

Snippet: <ul aria-labelledby="webchat__toaster__header__zos5z" class="webchat__toaster__list" id="webchat__toaster__list__vo8ez" role="region"></ul>

 

How to fix:

Fix all of the following:

  Invalid ARIA attribute value: aria-labelledby="webchat__toaster__header__zos5z"

 

Environment: Chrome version 80.0.3987.132

 

====

 

This accessibility issue was found using Accessibility Insights for Web 2.15.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

 

Recommendations:

Refer below link which is repository of bug fixes code snippets:

https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

 

MAS Reference:

 

MAS 4.1.1:Parsing

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={2e2a1540-aaf7-48a0-a53e-8f10cf309999}

 

MAS 4.1.2:

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Comment by compulim** [comment]
![image](https://user-images.githubusercontent.com/1622400/79015662-1898e800-7b22-11ea-9632-3deb714ce023.png)

![image](https://user-images.githubusercontent.com/1622400/79015752-4716c300-7b22-11ea-93c4-320dc20fdfb1.png)

![image](https://user-images.githubusercontent.com/1622400/79015816-66155500-7b22-11ea-8729-9b06fce89ddb.png)

We should remove `aria-labelled-by` by when the element it is referencing to is missing.
